cvat/apps/engine/views.py

```diff
@@ -492,9 +502,12 @@ def export_backup(self, request, pk=None):
             serializers=[ProjectFileSerializer, OpenApiTypes.NONE],
             resource_type_field_name=None
         ),
+        # TODO: for some reason the code generated by the openapi generator from schema with different serializers
+        # contains only one serializer, need to fix that.
```

Comment:
Probably, then new protocol should be described in the documentation somewhere. At least, in the endpoint description.